homeassistant/components/atome/sensor.py

```diff
@@ -0,0 +1,106 @@
+"""Linky Atome."""
+import logging
+import voluptuous as vol
+
+from datetime import timedelta
+from pyatome.client import AtomeClient
+from pyatome.client import PyAtomeError
+
+from homeassistant.const import (
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    CONF_NAME,
+    DEVICE_CLASS_POWER,
+    POWER_WATT,
+)
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+
```

Comment:
Remove all the blank lines between imports from the same group, standard library, 3rd party and homeassistant. It's just these three groups that should be separated by a blank line.